### PR TITLE
Update configuration page and add deny-chunk-requests to the optimization page

### PR DESCRIPTION
**Pull request by phit**
In prep for https://github.com/SpongePowered/SpongeForge/issues/2003
Also updated the rest of the missing entries that were added to the global.conf recently and some time ago

@Zidane gimme config nowᵖˡᵉᵃˢᵉ

### CODE REVIEW

File: source/server/management/performance-tweaks.rst [hunk 1]
```diff
@@ -137,3 +137,22 @@ One example of this is baby animals. Normally, they take 20 minutes to grow into
...
+.. code-block:: none
+
+    world {
+        # If enabled, any request for a chunk not currently loaded will be denied (exceptions apply for things like world gen and player movement). 
+        # Note: As this is an experimental setting for performance gain, if you encounter any issues then we recommend disabling it.
+        deny-chunk-requests=false
+    }
+
+When this option is enabled, mods requesting areas to be loaded to perform various tasks will be denied. Only players
+and specific world generation calls will be allowed to load new chunks.
```
Comment (ST-DDT) [inline source/server/management/performance-tweaks.rst]: Please add a new line at the end of file.

**Review by ST-DDT — COMMENTED** [review]
Is this PR WIP (as your request hints) or will it be extended later on?

**Comment by phit** [comment]
https://github.com/SpongePowered/SpongeForge/issues/2003 got merged and is live, so I've updated to latest config

let me know when its ready to merge and ill squash the commits

**Review by ST-DDT — APPROVED** [review]
Maybe we should remove the trailing whitespaces to reduce the total diff. But thats just a minor thing. Its okay if it consistent with past/future commits.

**Comment by phit** [comment]
I added trailing spaces to the actual config, so all future changes will have those to. Its nice when you copy parts of the config, just need to remove the line break and you have the sentences properly spaced out.

**Comment by Inscrutable** [comment]
I'm happy with it, as it's pretty much 1:1 with the current Sponge config file. Any further cleanup will have to happen there anyway. By the way, you don't have to worry about squashing. It automatic at this end when kronk pull lever.

**Pull request merged**